### PR TITLE
[ABACus-51] Test all the things. Part 2: ingest testing

### PR DESCRIPTION
**Pull request by JTR-USAF**
image

### CODE REVIEW

**Review by DL-USAF — APPROVED** [review]

File: src/tests/test_ingest_interactions.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
+import pytest
+
+from datawave_cli.utilities import pods
+from datawave_cli.utilities.utilities import Retry
+from datawave_cli.ingest_interactions import (check_app_statuses, get_mapreduce_statuses, get_accumulo_appstates,
+                                              check_for_file, copy_file_to_pod, check_for_required_cmds, main)
```
Comment (JTR-USAF) [inline src/tests/test_ingest_interactions.py:6]: I hate this. Fix it.

File: src/datawave_cli/ingest_interactions.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 from datawave_cli.utilities.utilities import setup_logger, Retry
 
 
-@Retry(time_limit_min=3)
+@Retry(time_limit_min=.5)
```
Comment (JTR-USAF) [inline src/datawave_cli/ingest_interactions.py:18]: Revert this to 3 minutes. Or else.

File: src/tests/test_ingest_interactions.py (hunk 1 above)
Comment (JTR-USAF) [reply src/tests/test_ingest_interactions.py:6]: And if you even THINK about `import *` we will have you kicked off the team!

File: src/datawave_cli/ingest_interactions.py (hunk 2 above)
Comment (JTR-USAF) [reply src/datawave_cli/ingest_interactions.py]: Took me two tries, but I fixed it!

**Review by DL-USAF — APPROVED** [review]

**Review by NC-USAF — APPROVED** [review]

**Pull request merged**